app/views/games/new.html.erb

```diff
@@ -0,0 +1,8 @@
+<%= simple_from_for (@game) do |f|%>
+  <%= f.input :name %>
+  <%= f.input :category %>
+  <%= f.input :description %>
+  <%= f.input :location %>
```

Comment:
`<%= f.input :category, :collection => %w[#Category of games that are in the validation] %>`
I think this needs to be like a dropdown list of the categories we have in the game model (look at the validation)

Reply:
Okay I will look at this thoroughly tonight Aswell!